ethcore/types/src/transaction/transaction.rs

```diff
@@ -439,9 +430,6 @@ impl From<SignedTransaction> for UnverifiedTransaction {
 impl SignedTransaction {
 	/// Try to verify transaction and recover sender.
 	pub fn new(transaction: UnverifiedTransaction) -> Result<Self, parity_crypto::publickey::Error> {
-		if transaction.is_unsigned() {
```

Comment:
the aforementioned type safety is here, `SignedTransaction::new` will not accept unsigned transactions after #11335